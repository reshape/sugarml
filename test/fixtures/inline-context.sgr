head:title foo
body.bar
