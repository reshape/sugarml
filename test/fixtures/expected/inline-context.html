<head><title>foo</title></head><body class="bar"></body>
